src/Instagram/Provider.php

```diff
@@ -114,8 +114,9 @@ protected function generateSignature($endpoint, array $params)
         foreach ($params as $key => $val) {
             $sig .= "|$key=$val";
         }
-        $signing_key = $this->clientSecret;
 
-        return hash_hmac('sha256', $sig, $signing_key, false);
+        $signingKey = $this->clientSecret;
+
+        return hash_hmac('sha256', $sig, $signingKey, false);
```

Comment:
could we just put the client secret straight into the call?
```suggestion
        return hash_hmac('sha256', $sig, $this->clientSecret, false);
```

Reply:
See 00f70ffa95a2fb1471d5129fc043595b11dfe524